### PR TITLE
chisel: add head

### PR DESCRIPTION
**Pull request by Ashton-W**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/chisel.rb [hunk 1]
```diff
@@ -4,6 +4,10 @@ class Chisel < Formula
   url "https://github.com/facebook/chisel/archive/1.4.0.tar.gz"
   sha256 "94d695cbc24343f8002c2b68a68bcc00601ab161ef24d16ee0718c93a2889493"
 
+  head do
```
Comment (DomT4) [inline Formula/chisel.rb]: You don't need the block format of `head do` unless you're doing something `head`-exclusive to it, such as resources or dependencies that apply to `head` but not `stable`. In this case, just:
```ruby
url "https://github.com/facebook/chisel/archive/1.4.0.tar.gz"
sha256 "94d695cbc24343f8002c2b68a68bcc00601ab161ef24d16ee0718c93a2889493"
head "https://github.com/facebook/chisel.git"
```
is fine.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/c8e6bced37ada1f91523fc7217e87c84b584f8f5. Thank you for your contribution to Homebrew @Ashton-W; we appreciate it! :smiley_cat:

**Comment by BiggerGao** [comment]
Have you fixed it for brew?I update the chisel version to 1.4 in brew but pjson still doesn't work and I get the same error.

**Comment by Ashton-W** [comment]
@BiggerGao I think you meant to comment on an issue in the chisel repo. Anyway, install with `brew install chisel --HEAD`

**Pull request merged**